src/utils/randomId/index.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import type { Random } from "./index.flow";
+
+const randomId: Random = (name: string) => {
+  return name + Math.floor(Math.random() * 1000000000) + Math.floor(Math.random() * 1000000000);
```

Comment:
😅 won't be `thousand` enough?

Reply:
Also, plz `randomID` 🙏

Reply:
1000 is a length of 3, that's not sufficient I had it originally on that, but after running a benchmark, it returned match multiple times on 1000 items. So I cracked that up, but maybe we don't need an addition of extra random number in that case?

Also, randomID is not proper camel-case